SearchImplem/City.swift

```diff
@@ -10,13 +10,10 @@ import Foundation
 
 class City: NSObject {
     
-    let name: String
-    let country: String
-    let adminName: String
+    var name: String?
+    var country: String?
+    var adminArea: String?
+    var coordinates: (Double, Double)?
```

Comment:
Use CLLocationCoordinate2D. or use 2 variables for latitude and logitude

Reply:
used CLLocationCoordinates